openapi/server.yml

```
@@ -0,0 +1,285 @@
+                    nullable: true
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+    get:
+      summary: Check Store Health
+      description: |
+        Checks whether we can connect to the underlying data store.
+        
```

yamllint

[error] trailing spaces

(trailing-spaces)